OnBoardingCalculator/app/src/main/res/values/strings.xml

```diff
@@ -17,4 +17,8 @@
     <string name="btn_text_divide">/</string>
     <string name="btn_text_multiply">X</string>
     <string name="btn_text_clear">CLEAR</string>
+    <string name="error_msj">Error: No se debe dividir por cero.</string>
```

Comment:
pls try it to put all the message in english

Reply:
Done!